app/models/enterprise.rb

```diff
@@ -199,7 +189,7 @@ def self.distinct_count
   end
 
   def activated?
-    confirmed_at.present? && sells != 'unspecified'
+    contact.confirmed? && sells != 'unspecified'
```

Comment:
Ok, well given our discussion above, if we add the migration to reset `sells` for any enterprises that don't have a confirmed `email`, we can probably just drop the `user.confirmed?` check here altogether?